build/v1/types.go

```diff
@@ -1016,6 +1022,27 @@ type ImageChangeTrigger struct {
 	Paused bool `json:"paused,omitempty" protobuf:"varint,3,opt,name=paused"`
 }
 
+// ImageChangeTriggerStatus tracks both the latest resolved status of the ImageChangeTrigger policy if it is
+// specified in the BuildConfigSpec.Triggers struct
+type ImageChangeTriggerStatus struct {
+	// lastTriggeredImageID contains the resolved DockerImage sha from the ImageStreamTag referenced in the
+	// ImageChangeTrigger.From field when that trigger resulted in a Build being initiated.
+	LastTriggeredImageID string `json:"lastTriggeredImageID,omitempty" protobuf:"bytes,1,opt,name=lastTriggeredImageID"`
+
+	// from is whichever ImageStreamTag, either an explicitly cited ImageChangeTrigger ImageStreamTag in the spec,
```

Comment:
sentence seems to be missing a word or words

Reply:
yep see below